example/src/screens/BasicFullscreenHandling.tsx

```diff
@@ -103,16 +97,19 @@ const styles = StyleSheet.create({
     flex: 1,
     alignItems: 'center',
     justifyContent: 'center',
-    backgroundColor: 'black',
+    backgroundColor: 'white',
+    padding: 20,
```

Comment:
Added white background and a bit of padding to make it more visual when the player becomes fullscreen.